docs/styling/theming.md

```diff
@@ -43,30 +97,56 @@ rx.flex(
 )
 ```
 
-### Specific Shades of Palettes
+### Shades
 
-To access a specific shade of color from the theme, you can use the `rx.color`. When switching to light and dark themes, the color will automatically change.
+Sometime you may want to use a specific shade of a color from the theme. This is recommended vs using a hex color directly as it will automatically change when the theme changes appearance change from light/dark.
 
-Shades can be accessed by using the color name and the shade number. The shade number ranges from 1 to 12. Additionally, they can have their alpha value set by using the `True` parameter it defaults to `False`.
+
+To access a specific shade of color from the theme, you can use the `rx.color`. When switching to light and dark themes, the color will automatically change. Shades can be accessed by using the color name and the shade number. The shade number ranges from 1 to 12. Additionally, they can have their alpha value set by using the `True` parameter it defaults to `False`. A full list of colors can be found [here](https://www.radix-ui.com/colors).
 
 ```python demo
 rx.flex(
     rx.button(
         "Hello World",
         color=rx.color("grass", 1),
-        background_color=rx.color("grass", 12),
+        background_color=rx.color("grass", 7),
         border_color=f"1px solid {rx.color('grass', 1)}",
     ),
-    rx.button(
-        "Hello World",
-        color=rx.color("grass", 1, True),
-        background_color=rx.color("grass", 12, True),
-        border_color=f"1px solid {rx.color('grass', 1, True)}",
-    ),
     spacing="2"
 )
 ```
 
+```python eval
+rx.table.root(
+    rx.table.header(
+        rx.table.row(
+            rx.table.column_header_cell(rx.code("Name")),
+            rx.table.column_header_cell("Type"),
+            rx.table.column_header_cell("Description"),
+        ),
+    ),
+    rx.table.body(
+        rx.table.row(
+            rx.table.row_header_cell(rx.code("color")),
+            rx.table.cell("Str"),
+            rx.table.cell("The color to use. Can be any color from the theme."),
+        ),
+        rx.table.row(
+            rx.table.row_header_cell(rx.code("shade")),
+            rx.table.cell('1 - 12'),
+            rx.table.cell("The shade of the color to use. Defaults to 7."),
```

Comment:
We should probably link to this https://www.radix-ui.com/colors/docs/palette-composition/understanding-the-scale

Reply:
i was thinking that specifically linking to the "Understanding the Scale" page would help people decode the why behind the numbers